src/display/editor/annotation_editor_layer.js

```diff
@@ -103,6 +103,12 @@ class AnnotationEditorLayer {
       this.enableClick();
     }
     this.#uiManager.unselectAll();
+
+    if (mode === AnnotationEditorType.FREETEXT) {
+      this.div.classList.add("freeTextEditing");
+    } else {
+      this.div.classList.remove("freeTextEditing");
+    }
```

Comment:
This could be shorter/simpler written as:
```suggestion
    this.div.classList.toggle("freeTextEditing", mode === AnnotationEditorType.FREETEXT);
```